src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -60,6 +61,14 @@ public virtual ICommandExecutive CommandExecutive
         }
 
         /// <summary>
+        /// Return logger for sending information to Dynamo console
+        /// </summary>
+        public ILogger Logger
+        {
+            get { return dynamoModel.Logger; }
```

Comment:
Similar but a bit different. I was proposing an `INotifier` interface which an extension could implement to create notifications to show up in the notifications bar. But this is a good start. Extensions definitely need _some_ logging capability.